### ISSUE TITLE
Nightmare Sigils affixes broken after patch 1.3.1 (All Platforms)

### ISSUE THREAD

**Issue by zbee** [aeon0/d4lf#165]
With 1.3.1 monster levels are now included in Sigils, an need cleaned from the affix reads.

```
[4.1.2 2024-02-01 14:29:10,748] INFO Retry item detection
[4.1.2 2024-02-01 14:29:11,858] WARNING Could not find affix: [cleaned]: monster level �s, [raw]: monster level: 85 �dungeon affixes
[4.1.2 2024-02-01 14:29:13,072] INFO Retry item detection
[4.1.2 2024-02-01 14:29:14,036] WARNING Could not find affix: [cleaned]: monster level you deal more physical damage., [raw]: monster level: 86 you deal 15% more physical damage.
[4.1.2 2024-02-01 14:29:15,220] INFO Retry item detection
[4.1.2 2024-02-01 14:29:16,150] WARNING Could not find affix: [cleaned]: monster level you find more gold., [raw]: monster level: 88 you find 30% more gold.
```

[Here are several logged failed sigils](https://github.com/aeon0/d4lf/files/14132525/failed_screenshots.zip)

**Comment by zbee** [comment]
Assumedly it would be in the same place as the [Revives Allowed hAndling](https://github.com/aeon0/d4lf/blob/703e469d67692d7b31a1e5280c6d7047cac1b0de/src/item/descr/find_affixes.py#L93) (mostly a note to self if I PR it)

**Comment by aeon0** [comment]
Yes, the addition of monster level messed up the current implementation. I have fixed it on current master and will be in in next release.

**Comment by aeon0** [comment]
fixed in 4.1.3
